### PR TITLE
Remove legacy bodypart id from covers

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
SUMMARY: None

#### Purpose of change

Clean up json

#### Describe the solution

Replace legacy id by new ids
Load `sided` from json
Remove legacy code

#### Testing
Load old save without error
Wear a watch withtout error
Change side on the watch > the watch is on the other hand

### CODE REVIEW

**Comment by SirPendrak** [comment]
If quiver had 
"covers": [ "LEG_EITHER" ],
and now it has 
"covers": [ "leg_l", "leg_r" ], 
Doesnt it mean that now quiver covers both legs at the same time?

**Comment by Fris0uman** [comment]
Looking at the code LEGS and LEG_EITHER do the same thing: set leg_l and leg_r

**Comment by Fris0uman** [comment]
Ah no, it's not the same X_EITHER makes `sided` true. Well I just need to change how `sided` is loaded

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -1801,9 +1811,7 @@
     "turns_per_charge": 30,
     "revert_to": "mask_h20survivor",
     "use_action": { "type": "transform", "menu_text": "Turn off", "msg": "Your %s deactivates.", "target": "mask_h20survivor" },
-    "covers": [ "eyes", "mouth" ],
-    "environmental_protection": 16,
-    "encumbrance": 20
```
Comment (SirPendrak) [inline data/json/items/tool_armor.json]: i believe it was intended for this thing to have lower encumbrance when turned on

File: data/json/items/tool_armor.json [hunk 2]
```diff
@@ -1851,9 +1858,7 @@
     "turns_per_charge": 30,
     "revert_to": "mask_h20survivorxl",
     "use_action": { "type": "transform", "menu_text": "Turn off", "msg": "Your %s deactivates.", "target": "mask_h20survivorxl" },
-    "covers": [ "eyes", "mouth" ],
-    "environmental_protection": 16,
-    "encumbrance": 20
```
Comment (SirPendrak) [inline data/json/items/tool_armor.json]: Same there

File: data/json/items/tool_armor.json (hunk 1 above)
Comment (Fris0uman) [reply data/json/items/tool_armor.json]: What do you mean? I did,'t change the values, only the name of the bodypart_id

Comment (SirPendrak) [reply data/json/items/tool_armor.json]: You did remove encumbrance value of "on" version of divemask. And before it had 20 enc. when on and 30 when "off" (have "varsize" in mind too). Now "on" version will have the same enc. and better env. protection.

Comment (Fris0uman) [reply data/json/items/tool_armor.json]: Look at the changes again I have only edited the id. The encumbrance thing has been edited by another PR and it showed here when I rebased this against master.

**Pull request merged**